project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject clj-firmata "2.1.2-SNAPSHOT"
+(defproject clj-firmata "2.1.2"
```

Comment:
Bumping the version is a release activity

Reply:
Commited a fixup without the version bump.